rpc/tendermint/test/tests.go

```diff
@@ -251,7 +253,59 @@ func asEventDataTx(t *testing.T, eventData txs.EventData) txs.EventDataTx {
 	return eventDataTx
 }
 
-func doNothing(eventId string, eventData txs.EventData) (bool, error) {
+func doNothing(_ string, _ txs.EventData) (bool, error) {
 	// And ask waitForEvent to stop waiting
 	return true, nil
 }
+
+func testSubscribe(t *testing.T) {
```

Comment:
Note this is not a *_test.go file.

It's following the same pattern as other methods in here, it's the testing method that the real one in https://github.com/silasdavis/eris-db/blob/f632852bb6e032292b9cbadb427b03c807ddb018/rpc/tendermint/test/client_ws_test.go#L241 calls.

Aside from following that pattern, the may reason I have it here is because go-wire doesn't work from *_test.go files it would seem. Something to do with the global var interface registration.